### PR TITLE
Adding a new planck layout

### PR DESCRIPTION
**Pull request by rennerom**
## Description

Adding a new layout for Planck

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by rennerom** [comment]
Got ahead of myself with my Preonic keymap. I pulled out those commits and just left the Planck keymap files as part of this PR. I'll submit a separate PR with my Preonic keymap later. 

Sorry for the confusion!

File: keyboards/planck/keymaps/rennerom/config.h [hunk 1]
```diff
@@ -0,0 +1,41 @@
+#pragma once
```
[the remark this answers is not in the document]
Comment (rennerom) [reply keyboards/planck/keymaps/rennerom/config.h:4]: Added!

**Comment by rennerom** [comment]
Added gpl2+, removed bloat.

File: keyboards/planck/keymaps/rennerom/keymap.c [hunk 2]
```diff
@@ -0,0 +1,179 @@
+/*
```
Comment (drashna) [inline keyboards/planck/keymaps/rennerom/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Or 

https://github.com/qmk/qmk_firmware/blob/6b299fab6e6a3143c061b9ea005253361fcd524b/data/templates/base/keyboard.c#L1-L15

Comment (rennerom) [reply keyboards/planck/keymaps/rennerom/keymap.c]: Done!

**Review by drashna — APPROVED** [review]

File: keyboards/planck/keymaps/rennerom/config.h [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+#define TAPPING_TERM 200
+
+/*
+ * MIDI options
+ */
+
+/* Prevent use of disabled MIDI features in the keymap */
+//#define MIDI_ENABLE_STRICT 1
+
```
Comment (fauxpark) [inline keyboards/planck/keymaps/rennerom/config.h:25]:
```suggestion
```

File: keyboards/planck/keymaps/rennerom/config.h [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+   - MIDI notes can be added to the keymap
+   - Octave shift and transpose
+   - Virtual sustain, portamento, and modulation wheel
+   - etc.
+*/
+//#define MIDI_ADVANCED
+
+/* override number of MIDI tone keycodes (each octave adds 12 keycodes and allocates 12 bytes) */
+//#define MIDI_TONE_KEYCODE_OCTAVES 2
+
```
Comment (fauxpark) [inline keyboards/planck/keymaps/rennerom/config.h:42]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**